gradle/libs.versions.toml

```diff
@@ -231,3 +230,4 @@ opencsv = [
 [plugins]
 gradleWrapperUpgrade = { id = "org.gradle.wrapper-upgrade", version = "0.11.4" }
 koverPlugin = { id = "org.jetbrains.kotlinx.kover", version = "0.8.2" }
+screenshot = { id = "com.android.compose.screenshot", version.ref = "screenshot"}
```

Comment:
For this to work with our buildSrc, you might need to add it as library, declare it in buildSrc (the same pattern for the other convention plugins)

Reply:
## ivy.compose.preview.testing.gradle.kts

![image](https://github.com/user-attachments/assets/88efd308-c227-4424-b7b0-e34e08b148bc)

I try implement as your suggestion but even though I declared it in gradle.properties, it always returns this error

## Error

![image](https://github.com/user-attachments/assets/a12425ec-aeb4-457c-aeed-7d0109a42970)

I still don't have any ideas for this problem, wdyt? 
@ILIYANGERMANOV

Reply:
Do we have this in thing in "gradle.properties"? @isaacnguyen0809

Reply:
Yes, we already have it

Reply:
If i add "plugin id("com.android.compose.screenshot")" to sub module then it works fine, maybe when buildSrc run before gradle.properties apply. Just speculation and I'm not sure

Reply:
Idk, good theory! Did you try:
1. Add the plugin as implementation in buildSrc (you need to find the plugin as a library)
2. Follow the pattern for the other plugins

If it doesn't work, we can do it as you suggested. It's not great but it works. In general, I want to make creating new screen modules as easy as possible

Reply:
I tried and unfotunately, it doesn't work, error is like the photo i sent above. So i think will do as i suggested.

But I will leave a comment or todo so that after there is a stable version of Compose preview screenshot tesing(we don't need experimentalProperties anymore), the implementation as you said will not be a problem. We will refactor the code later.